fsharp-backend/src/BackendOnlyStdLib/HttpClient.fs

```diff
@@ -275,6 +287,8 @@ let makeHttpCall
       return Error { url = url; code = code; error = e.Message }
   }
 
+/// Wraps around `makeHttpCall`, handling redirects
+/// WHATISTHIS maybe a rename to makeHttpCallWithRedirects
```

Comment:
The intent here was to have `httpCall` being the important function. `makeHttpCall` is intended to be `httpCall'` (http call prime), that is, the private function that does the actual work behind the function that we call. I'd rename `makeHttpCall`.